packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -296,6 +298,10 @@ export const Workbench: React.FunctionComponent<{
         setSelectedTime={setSelectedTime}
         onSelected={onActionSelected}
         onHighlighted={setHighlightedAction}
+        revealAttachment={attachment => {
```

Comment:
You probably want to `React.useCallback()` here to avoid a new function instance every time.